pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -228,10 +242,22 @@ loki.process "kubernetes_audit" {
 		source = "objectRef"
 	}
 
+	stage.structured_metadata {
+		values = {
+			"resource" = "",
+			"filename" = "",
+		}
+	}
+
+	stage.label_drop {
+		values = [
+			"filename",
```

Comment:
Don't you need to also drop `resource` there?

Reply:
Nope because I did not make it a label in the stage.labels underneath